yeonjae/programmers-2/가장큰수.js

```diff
@@ -0,0 +1,21 @@
+function solution(numbers) {
+  let answer = "";
+  let numToStringArr = numbers.map((data) => String(data));
+
+  answer = numToStringArr.sort((a, b) => {
+    return b + a - (a + b);
+  });
+
+  return answer.join("")[0] == 0 ? "0" : answer.join("");
+}
```

Comment:
- 풀이잘봤습니다
- 구글링하시면 저희가 기존까지 잘 지켰던 부분들은 잘지켜서 재풀이해주세요.
- let, const 부분과 == 연산자 사용 등 리팩토링이 전혀 되어 있지않있습니다.
```suggestion
function solution(numbers) {
  const numToStringArr = numbers.map((data) => String(data));
  const sortedArr = numToStringArr.sort((a, b) => {
    return b + a - (a + b);
  });

  return sortedArr[0] === 0 ? "0" : sortedArr.join("");
}
```

Reply:
이번에 검토가 부족했던것같습니다
좀더리팩토링에 신경쓰도록할게요!